src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -243,6 +243,10 @@
     <value>The '_revinclude:iterate={0}' search parameter has multiple target types. Please specify a target type.</value>
     <comment>_revinclude:iterate is a query parameter.</comment>
   </data>
+  <data name="IncludeRevIncludeInvalidTargetResourceType" xml:space="preserve">
+    <value>Invalid TargetResourceType : value cannot be empty.</value>
```

Comment:
nit: Can we simplify the error message - 'The target resource type cannot be empty'

Reply:
yes